AsyncSwift/Views/EventDetailView.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  EventDetailView.swift
+//  AsyncSwift
+//
+//  Created by Kim Insub on 2022/09/08.
+//
+
+import SwiftUI
+
+struct EventDetailView: View {
+
+    private let event: EventModel.Event
+    @StateObject var observed = Observed()
+
+    init(event: EventModel.Event) {
+        self.event = event
+    }
+
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: 0) {
+                customDivider
+                    .padding(.top, 10)
+                description
+                customDivider
+                information
+                Spacer()
+            }
+        }
+        .navigationTitle(event.detailTitle)
+        .confirmationDialog("", isPresented: $observed.showSheet, titleVisibility: .hidden) {
+            if let naverMapURL = URL(string: event.addressURLs.naverMapURL) {
+                Link("네이버 지도로 길 찾기", destination: naverMapURL)
+            }
+            if let kakaoMapURL = URL(string: "event.addressURLs.kakaoMapURL") {
+                Link("카카오맵으로 길 찾기", destination: kakaoMapURL)
+            }
+        }
+    }
+}
+
+private extension EventDetailView {
+
+    var description: some View {
+        VStack(alignment: .leading) {
+            Text(event.subject)
+                .fontWeight(.bold)
+                .font(.title3)
```

Comment:
```suggestion
                .font(.system(size: 20, weight: .bold))
```
font Weight가 중복되서 선언됩니다!

Reply:
Title3 폰트 사이즈에 대응하기 위해 사이즈를 .title3로 가지고 오는것이고
웨이트는 .title3가 regular 이기 때문에 그렇습니다